experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -306,4 +327,63 @@ export class NodeSDK {
         .then(() => {})
     );
   }
+
+  private configureLoggerProviderFromEnv(
+    envWithoutDefaults: ENVIRONMENT
+  ): void {
+    const logExportersList = envWithoutDefaults.OTEL_LOGS_EXPORTER;
```

Comment:
I try to discourage using the environment handling form `@opentelemetry/core`. It's more complex than it needs to be and we're planning to remove it once we transition to SDK 2.0. 

Replacing this with `process.env` would be more future-proof.